cpp/test/CMakeLists.txt

```diff
@@ -194,7 +194,7 @@ if(BUILD_PRIMS_TESTS)
     prims/sparse/csr_to_dense.cu
     prims/sparse/csr_transpose.cu
     prims/sparse/distance.cu
-    prims/sparse/knn.cu
+          prims/sparse/knn.cu
```

Comment:
is the indent a little weird here? in the diff it renders like it has a different indent

Reply:
clion does this for some reason. I have to keep remembering to go back and fix these.